src/lambda/wizards/samDeployWizard.ts

```diff
@@ -113,33 +113,38 @@ class DefaultSamDeployWizardContext implements SamDeployWizardContext {
         return choices[0].uri
     }
 
-    public async promptUserForRegion(regionProvider: RegionProvider,
-                                     initialRegionCode: string): Promise<string | undefined> {
+    public async promptUserForRegion(
+        regionProvider: RegionProvider,
+        initialRegionCode: string
+    ): Promise<string | undefined> {
         const logger = getLogger()
         const regionData = await regionProvider.getRegionData()
 
-        const quickPick = picker.createQuickPick<vscode.QuickPickItem>({
-            options: {
-                title: localize('AWS.samcli.deploy.region.prompt',
-                                'Which AWS Region would you like to deploy to?'),
-                value: initialRegionCode || '',
-                matchOnDetail: true,
-                ignoreFocusOut: true,
-            },
-            items: regionData.map(r => ({
-                label: r.regionName,
-                detail: r.regionCode,
-                // this is the only way to get this to show on going back
-                // this will make it so it always shows even when searching for something else
-                alwaysShow: r.regionCode === initialRegionCode,
-                description: r.regionCode === initialRegionCode ? localize('AWS.samcli.deploy.region.previousRegion',
-                                                                           'Selected Previously')
-                                                                : ''
-            })),
-            buttons: [
-                vscode.QuickInputButtons.Back
-            ],
-        })
+        const quickPick = picker.createQuickPick<vscode.QuickPickItem>(
+            {
+                options: {
+                    title: localize('AWS.samcli.deploy.region.prompt',
```

Comment:
It's fine to include the braces on the same line like in your previous iteration. This comment was about the `localize` call, which should be:

```typescript
title: localize(
    'AWS.samcli.deploy.region.prompt',
    'Which AWS Region would you like to deploy to?'
)
```

Reply:
Done